benchmarking/README.md

```diff
@@ -88,60 +93,128 @@ git clone https://github.com/sambanova/ai-starter-kit.git
+
+    - ```throughput_token_per_s``` bar plot: This plot shows the total tokens generated per second per batch. One should see good agreement between the client and server-side metrics. This metric represents the total number of tokens generated per second, which is the same as the previous metric for batch size = 1. However, for batch size > 1, it is estimated as the average of ```output_token_per_s_per_request * batch_size_used``` for each batch, to account for more tokens being generated due to concurrent requests being served in batch mode.
+
+#### Custom Performance Evaluation
+
+This option allows you to evaluate the performance of the selected LLM on your own custom dataset. The interface should look like this:
+
+![Custom Performance Evaluation](./imgs/custom_performance_eval.png)
 
-In order to use this functionality, please follow the steps below:
+1. Prep your dataset
 
-1. Introduce the LLM model
+- The dataset needs to be in `.jsonl` format - these means a file with one JSON object per line.
+- Each JSON object should have a `prompt` key with the value being the prompt you want to pass to the LLM.
+  - You can use a different keyword instead of `prompt`, but it's important that all your JSON objects use the same key
 
-Under the section `Configuration`, users need to introduce the LLM model that will be used for the performance evaluation process. If it's a CoE model, add a `COE/` prefix to the name (for example, `COE/Meta-Llama-3-8B-Instruct`). If you're not sure about the name of the model/expert you want to choose, please go to the model card in SambaStudio and search for the model/expert name.
+2. Enter the dataset path
 
-2. Choose parameter values
+- The entered path should be an absolute path to your dataset.
+  - For example: `/Users/johndoe/Documents/my_dataset.jsonl`
 
-Different LLM parameters are available for experimentation, directly related to the previously introduced LLM. The app provides toggles and sliders to facilitate the configuration of all these parameters. Users can use the default values or modify them as needed.
+3. Enter a model name
 
-- Number of input tokens: average number of input tokens. Default value: 1000.
-- Number of output tokens: average number of output tokens. Default value: 1000.
-- Number of total requests: maximum number of completed requests. Default value: 32. 
-- Number of concurrent workers: number of concurrent workers. Default value: 1. For testing [batching-enabled models](https://docs.sambanova.ai/sambastudio/latest/dynamic-batching.html), this value should be greater than the largest batch_size one needs to test. The typical batch sizes that are supported are 1,4,8 and 16.
-- Timeout: time when the process will stop. Default value: 600 seconds
+  - If the model specified in the `ai_starter-kit/.env` file is not a CoE model, you can skip this step.
```

Comment:
same suggestion as the one for synthetic perf eval

Reply:
✅